src/misc/PointcloudRasteriser.cpp

```diff
@@ -8,15 +8,16 @@
 #include <random>
 
 namespace roofer {
-
+  
   void RasterisePointcloud(
     PointCollection& pointcloud,
     LinearRing& footprint,
     ImageMap& image_bundle,
     // RasterTools::Raster& heightfield,
     float cellsize
   ) {
-    bool use_footprint = true;
+    //TODO: this is always true?
```

Comment:
Yes at the moment. This is because this code was transferred from another code base, and it needs some more refactoring to expose these options.